dash/development/_r_components_generation.py

```diff
@@ -25,8 +25,8 @@
         )
 
     component$props <- filter_null(component$props)
-    
-    structure(component, class = c('dash_component', 'list'))    
```

Comment:
Not sure why this trailing whitespace wasn't caught by CI earlier... it's going to generate diffs in all the component packages that use it.